src/browser/animations.js

```diff
@@ -241,14 +244,9 @@ function AnimationHandler(desiredInterval) {
                                 });
                             }
                             hwndToId[hwnd] = bw.id;
-                            wt.setWindowPos(hwnd, {
-                                x: newBounds.x,
-                                y: newBounds.y,
-                                w: newBounds.width,
-                                h: newBounds.height,
-                                flags: windowTransaction.flag.noZorder + windowTransaction.flag.noActivate
-                            });
+                            wt.setWindowPos(hwnd, { x, y, w: width, h: height, flags });
                         } else {
+                            const newBounds = clipBounds({ x, y, width, height }, bw);
```

Comment:
Since you're adding `clipBounds` here, you should add it to all the other calls too:
```
setWindowPlacement()
setWindowPos()
```